src/components/UI/ToastMessage/ToastMessage.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Snackbar, Button, IconButton } from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
```

Comment:
I don't see this is been used anywhere. Do we need this?

Reply:
I was planning on using them. Will remove it and checkin.